kernel/src/analysis/javascript.rs

```diff
@@ -133,7 +133,7 @@ lazy_static! {
     }
         "#,
         );
-        let mut rt = JsRuntimeForSnapshot::new(Default::default(), Default::default());
+        let mut rt = JsRuntimeForSnapshot::new(RuntimeOptions::default(), Default::default());
```

Comment:
Readability change. See this [clippy lint](https://rust-lang.github.io/rust-clippy/master/index.html#/default_trait_access).